.circleci/config.yml

```diff
@@ -1,51 +1,130 @@
-version: 2
+version: 2.1 # Set the CI version.
 
+
+# We use orbs to provide some common ruby and node tasks and shorten our config.
+# Learn more about orbs: https://circleci.com/orbs/
+orbs:
+  ruby: circleci/ruby@0.1.2
+  node: circleci/node@1.1.6
+
+
+# Yaml References enable us to DRY our our config by sharing variables across multiple jobs.
```

Comment:
What is DRY? ... does it just mean shorten/simplify?

Reply:
Woops! DRY stand for "Don't Repeat Yourself" I should also fix that duplicate `our` typo after 😅

Reply:
Ohhhhh